src/main/java/org/example/automation/amazon/pageobject/pages/DeliverLocationPage.java

```diff
@@ -0,0 +1,55 @@
+package org.example.automation.amazon.pageobject.pages;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.SelenideElement;
+import org.openqa.selenium.By;
+
+import static com.codeborne.selenide.Selenide.$;
+
+public class DeliverLocationPage {
+    public DeliverLocationPage enterZipCode(String zipCode) {
+        SelenideElement fieldUsZipCode = $(By.id("GLUXZipUpdateInput")).shouldBe(Condition.visible);
```

Comment:
Not critical now, but maybe in the future it will be better to move this value $(By.id("GLUXZipUpdateInput")) as a variable. Because if you will be using this value in different places. If the value is changed, you will need to change it everywhere. Same for other